### PR TITLE
LIMS 4350 :Orders: Create: Orders Choose test plans without test units

### PR DESCRIPTION
**Pull request by omnia353**
I updated this PR to create order with 3 suborders using api
```
➜  1lims-automation git:(LIMS--4350) ✗ nosetests -vs --nologcapture --tc-file=config.ini ui_testing/testcases/basic_tests/test006_orders.py:OrdersTestCases.test096_choose_test_plans_without_test_units
2020-09-14 15:15:40.298 | INFO     | api_testing.apis.base_api:_get_authorized_session:46 - Get authorized api session.
2020-09-14 15:15:40.299 | INFO     | api_testing.apis.base_api:_get_authorized_session:47 - admin:admin
2020-09-14 15:15:40.890 | INFO     | api_testing.apis.base_api:_get_authorized_session:58 - session ID : Sz0DxoZIqqf_n9mCdzGXnyAScWlZO_uAmuztGTJAUvU .....
Orders: Create: Orders Choose test plans without test units ...         
2020-09-14 15:15:59.860 | INFO     | ui_testing.testcases.base_test:setUp:27 - Test case : test096_choose_test_plans_without_test_units
2020-09-14 15:16:06.071 | INFO     | ui_testing.pages.base_pages:wait_until_page_is_loaded:582 - wait until page is loaded
2020-09-14 15:16:06.122 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-09-14 15:16:06.834 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-09-14 15:16:07.527 | INFO     | api_testing.apis.test_unit_api:set_name_configuration_name_only:427 - set test unit configuration
2020-09-14 15:16:07.956 | INFO     | api_testing.apis.base_api:set_configuration:73 - status code: 1
2020-09-14 15:16:07.957 | INFO     | api_testing.apis.orders_api:set_configuration:489 - set order configuration
2020-09-14 15:16:08.369 | INFO     | api_testing.apis.base_api:set_configuration:73 - status code: 1
2020-09-14 15:16:08.370 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/orders/auto/generatedOrderNumber/?yearOption=1
2020-09-14 15:16:08.707 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:08.708 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/contacts
2020-09-14 15:16:09.003 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:09.004 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/articles
2020-09-14 15:16:09.115 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:09.116 | INFO     | api_testing.apis.article_api:get_formatted_article_with_formatted_material_type:207 - search for article with material type {'id': 1, 'text': 'Raw Material'}
2020-09-14 15:16:09.117 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/materialTypes
2020-09-14 15:16:09.213 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:09.215 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testUnits
2020-09-14 15:16:09.354 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:09.355 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testUnits/get/1601
2020-09-14 15:16:09.457 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:09.458 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans
2020-09-14 15:16:09.607 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:09.608 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans/get/90137
2020-09-14 15:16:09.727 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:09.728 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/materialTypes
2020-09-14 15:16:09.822 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:09.823 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testUnits
2020-09-14 15:16:09.950 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:09.951 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testUnits/get/1602
2020-09-14 15:16:10.094 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:10.094 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans
2020-09-14 15:16:10.238 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:10.238 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans/get/92720
2020-09-14 15:16:10.363 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:10.364 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/articles
2020-09-14 15:16:10.469 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:10.469 | INFO     | api_testing.apis.article_api:get_formatted_article_with_formatted_material_type:207 - search for article with material type {'id': 1, 'text': 'Raw Material'}
2020-09-14 15:16:10.470 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/materialTypes
2020-09-14 15:16:10.565 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:10.565 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testUnits
2020-09-14 15:16:10.699 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:10.700 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testUnits/get/1603
2020-09-14 15:16:10.823 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:10.823 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans
2020-09-14 15:16:10.958 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:10.959 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans/get/85342
2020-09-14 15:16:11.089 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:11.090 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/materialTypes
2020-09-14 15:16:11.179 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:11.180 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testUnits
2020-09-14 15:16:11.307 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:11.307 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testUnits/get/1604
2020-09-14 15:16:11.407 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:11.408 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans
2020-09-14 15:16:11.609 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:11.609 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans/get/61621
2020-09-14 15:16:11.769 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:11.769 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/articles
2020-09-14 15:16:11.876 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:11.876 | INFO     | api_testing.apis.article_api:get_formatted_article_with_formatted_material_type:207 - search for article with material type {'id': 1, 'text': 'Raw Material'}
2020-09-14 15:16:11.877 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/materialTypes
2020-09-14 15:16:11.983 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:11.984 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testUnits
2020-09-14 15:16:12.144 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:12.145 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testUnits/get/1605
2020-09-14 15:16:12.243 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:12.243 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans
2020-09-14 15:16:12.380 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:12.380 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans/get/10514
2020-09-14 15:16:12.587 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:12.588 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/materialTypes
2020-09-14 15:16:12.679 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:12.680 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testUnits
2020-09-14 15:16:12.822 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:12.823 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testUnits/get/1606
2020-09-14 15:16:12.925 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:12.925 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans
2020-09-14 15:16:13.082 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:13.082 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans/get/47983
2020-09-14 15:16:13.199 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:13.200 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/orders
2020-09-14 15:16:13.601 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:13.601 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/orders/suborder/?orderId=363&deleted=0
2020-09-14 15:16:13.724 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:13.725 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans/get/10514
2020-09-14 15:16:13.961 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:13.962 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans/get/47983
2020-09-14 15:16:14.113 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:14.114 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans/get/10514
2020-09-14 15:16:14.234 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:14.234 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans/get/47983
2020-09-14 15:16:14.356 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:14.356 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans/get/10514
2020-09-14 15:16:14.466 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:14.466 | INFO     | api_testing.apis.base_api:wrapper:130 - GET : https://automation.1lims.com/api/testPlans/get/47983
2020-09-14 15:16:14.596 | DEBUG    | api_testing.apis.base_api:wrapper:137 - status code: 1
2020-09-14 15:16:14.597 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-09-14 15:16:20.817 | INFO     | ui_testing.pages.orders_page:filter_by_order_no:138 - Filter by order no. : 427
2020-09-14 15:16:27.322 | INFO     | ui_testing.pages.base_pages:wait_until_page_is_loaded:582 - wait until page is loaded
2020-09-14 15:16:27.364 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-09-14 15:16:28.073 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-09-14 15:16:34.023 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-09-14 15:16:34.767 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-09-14 15:16:40.832 | DEBUG    | ui_testing.pages.base_pages:sleep_medium:47 - wait up to 2 sec
2020-09-14 15:16:48.511 | INFO     | ui_testing.testcases.basic_tests.test006_orders:test096_choose_test_plans_without_test_units:3240 - assert the order table has been updated
2020-09-14 15:16:54.096 | DEBUG    | ui_testing.pages.base_pages:sleep_small:43 - wait up to 1 sec
2020-09-14 15:16:57.270 | INFO     | ui_testing.pages.analyses_page:filter_by_order_no:22 - Filter by order number : 427
2020-09-14 15:16:57.270 | INFO     | ui_testing.pages.base_pages:open_filter_menu:90 - open Filter
2020-09-14 15:17:24.265 | INFO     | ui_testing.pages.base_pages:wait_until_page_is_loaded:582 - wait until page is loaded
2020-09-14 15:17:24.558 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-09-14 15:17:28.557 | INFO     | ui_testing.testcases.basic_tests.test006_orders:test096_choose_test_plans_without_test_units:3247 - assert the analysis table has been updated
2020-09-14 15:17:33.874 | DEBUG    | ui_testing.pages.base_pages:sleep_medium:47 - wait up to 2 sec
2020-09-14 15:17:38.155 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-09-14 15:17:49.517 | DEBUG    | ui_testing.pages.base_pages:sleep_medium:47 - wait up to 2 sec
2020-09-14 15:17:54.025 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-09-14 15:18:05.168 | DEBUG    | ui_testing.pages.base_pages:sleep_medium:47 - wait up to 2 sec
2020-09-14 15:18:10.161 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-09-14 15:18:16.089 | INFO     | ui_testing.testcases.base_test:tearDown:33 - go to dashboard page
2020-09-14 15:18:19.727 | INFO     | ui_testing.testcases.base_test:tearDown:35 - TearDown.     
ok

----------------------------------------------------------------------
Ran 1 test in 158.697s

OK


```

### CODE REVIEW

**Review by ghada-37 — APPROVED** [review]

File: ui_testing/testcases/basic_tests/test006_orders.py [hunk 1]
```diff
@@ -3078,3 +3078,63 @@ def test090_if_cancel_archive_order_no_order_suborder_analysis_will_archived(sel
...
+            if i == 2:
+                tstunits = test_units1
+            elif i == 1:
+                tstunits = test_units2
+            else:
+                tstunits = test_units3
+            analysis_data = self.order_page.get_child_table_data(index=i)
+            self.assertEqual(len(analysis_data), 2)
+            for testunit in analysis_data:
+                self.assertIn(testunit['Test Unit'].replace("'", ""), tstunits)
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test006_orders.py:3140]:
```suggestion
        self.info('create multiple testplan with same article of each suborder')
        created_data_list = []
        for _ in range(3):
            tp_1, tp_2, article = TestPlanAPI().create_multiple_test_plan_with_same_article()
            created_data = {'tp_1': tp_1,
                            'tp_2': tp_2,
                            'article': article,
                            'test_units': [tp_1['specifications'][0]['name'],
                                           tp_2['specifications'][0]['name']]}
            created_data_list.append(created_data)

        self.info('create new order.')
        order_no = self.order_page.create_new_order(article=created_data_list[0]['article'],
                                                    material_type='Raw Material',
                                                    test_plans=[created_data_list[0]['tp_1']['testPlanEntity']['name'],
                                                                created_data_list[0]['tp_2']['testPlanEntity']['name']],
                                                    set_tstunit=False, save=False)
        self.info('add two more suborders')
        for i in range(1, 3):
            self.order_page.create_new_suborder(material_type='Raw Material',
                                                test_plans=[created_data_list[i]['tp_1']['testPlanEntity']['name'],
                                                            created_data_list[i]['tp_2']['testPlanEntity']['name']],
                                                article_name=created_data_list[i]['article'], add_tstunit=False)

        self.order_page.save(save_btn='order:save_btn')
        order_id = self.order_page.get_order_id()
        suborders = self.orders_api.get_suborder_by_order_id(order_id)[0]['orders']
        analysis_no = []
        self.info('assert each suporder has one analysis number.')
        for suborder in suborders:
            self.assertEqual(len(suborder['analysis']), 1)
            analysis_no.append(suborder['analysis'][0])

        self.info('assert the 3 analysis numbers are unique for each suborder ')
        self.assertEqual(3, len(set(analysis_no)))

        self.orders_page.get_orders_page()
        self.orders_page.filter_by_order_no(filter_text=order_no)
        suborders_data = self.order_page.get_child_table_data(index=0)
        analysis_no_list = []
        for suborder in suborders_data:
            analysis_no_list.append(suborder['Analysis No.'])
        self.info('assert the analysis table has been updated')
        self.assertCountEqual(analysis_no, analysis_no_list)

        self.orders_page.navigate_to_analysis_active_table()
        self.analyses_page.filter_by_order_no(filter_text=order_no)
        analysis_rows = self.order_page.result_table()
        self.assertEqual(3, len(analysis_rows) - 1)

        for i in range(3):
            analysis_data = self.order_page.get_child_table_data(index=2-i)
            self.assertEqual(len(analysis_data), 2)
            analysis_tus = [ad['Test Unit'].replace("'", "") for ad in analysis_data]
            self.assertCountEqual(analysis_tus, created_data_list[i]['test_units'])
                
```

**Review by 0xIslamTaha — CHANGES_REQUESTED** [review]

File: ui_testing/pages/order_page.py [hunk 2]
```diff
@@ -155,9 +155,10 @@ def create_new_order(self, material_type='', article='', contact='', test_plans=
             for test_plan in test_plans:
                 self.set_test_plan(test_plan=test_plan)
 
-        for test_unit in test_units:
-            self.set_test_unit(test_unit=test_unit)
-            self.sleep_small()
+        if set_tstunit:
```
[the remark this answers is not in the document]
Comment (omnia353) [reply ui_testing/pages/order_page.py:158]: okay i will change it

File: api_testing/apis/orders_api.py [hunk 3]
```diff
@@ -163,12 +159,14 @@ def create_order_with_multiple_suborders(self, no_suborders=3, suborders_fields=
...
         suborders = []
+        sub_order_dict = {}
         for i in range(no_suborders):
-            sub_order_dict = {**suborders_common_data}
+            sub_order_dict = {** suborders_common_data}
             if len(suborders_fields) > i:
                 for dict_key in suborders_fields[i].keys():
                     sub_order_dict[dict_key] = suborders_fields[i][dict_key]
             suborders.append(sub_order_dict)
+            sub_order_dict = {}
```
Comment (0xIslamTaha) [inline api_testing/apis/orders_api.py:169]: there is no need for it. please remove it.

File: api_testing/apis/orders_api.py [hunk 4]
```diff
@@ -163,12 +159,14 @@ def create_order_with_multiple_suborders(self, no_suborders=3, suborders_fields=
             'testDatedateOption': {'year': test_date_arr[0], 'month': test_date_arr[1], 'day': test_date_arr[2]}
         }
         suborders = []
+        sub_order_dict = {}
```
Comment (0xIslamTaha) [inline api_testing/apis/orders_api.py:162]: there is no need for it. please remove it.

File: api_testing/apis/orders_api.py [hunk 5]
```diff
@@ -511,3 +509,26 @@ def set_contact_configuration_to_number_only(self):
         with open(config_file, "r") as read_file:
             payload = json.load(read_file)
         super().set_configuration(payload=payload)
+
+    def create_order_with_multiple_suborders_double_tp(self, no_suborders=3):
+        suborders = []
+        for i in range(no_suborders):
```
Comment (0xIslamTaha) [inline api_testing/apis/orders_api.py:515]: `for _ in range(no_suborders):`

File: api_testing/apis/orders_api.py [hunk 6]
```diff
@@ -511,3 +509,26 @@ def set_contact_configuration_to_number_only(self):
...
+            formatted_testPlan2 = {'id': int(tp_2['id']), 'name': tp_2['testPlanEntity']['name'], 'version': 1}
+            testplan_list = [formatted_testPlan1, formatted_testPlan2]
+            suborder['testPlans'] = testplan_list
+            suborder['selectedTestPlans'] = testplan_list
+            suborder['materialType'] = formatted_material_type
+            suborder['materialTypeId'] = formatted_material_type['id']
+            suborder['article'] = formatted_article
+            suborder['articleId'] = article_id
+            suborders.append(suborder)
+        return self.create_order_with_multiple_suborders(no_suborders=3, suborders_fields=suborders)
```
Comment (0xIslamTaha) [inline api_testing/apis/orders_api.py:534]: `return self.create_order_with_multiple_suborders(no_suborders=no_suborders, suborders_fields=suborders)`

File: api_testing/apis/test_plan_api.py [hunk 7]
```diff
@@ -363,4 +363,16 @@ def set_configuration(self):
         config_file = os.path.abspath('api_testing/config/test_plan.json')
         with open(config_file, "r") as read_file:
             payload = json.load(read_file)
-        super().set_configuration(payload=payload)
\ No newline at end of file
+        super().set_configuration(payload=payload)
+
+    def create_multiple_test_plan_with_same_article(self):
```
Comment (0xIslamTaha) [inline api_testing/apis/test_plan_api.py:368]: This could take no_of_test_plans and returns a list of the created test plans.

**Review by 0xIslamTaha — CHANGES_REQUESTED** [review]

File: api_testing/apis/orders_api.py (hunk 3 above)
Comment (ghada-37) [reply api_testing/apis/orders_api.py:169]: done

File: api_testing/apis/orders_api.py (hunk 4 above)
Comment (ghada-37) [reply api_testing/apis/orders_api.py:162]: done

File: api_testing/apis/orders_api.py (hunk 5 above)
Comment (ghada-37) [reply api_testing/apis/orders_api.py:515]: I don't get why I should replace i by -

File: api_testing/apis/orders_api.py (hunk 6 above)
Comment (ghada-37) [reply api_testing/apis/orders_api.py:534]: done

File: api_testing/apis/test_plan_api.py (hunk 7 above)
Comment (ghada-37) [reply api_testing/apis/test_plan_api.py:368]: done

**Review by 0xIslamTaha — APPROVED** [review]

**Pull request merged**